recipes/_configure_authentication.rb

```diff
@@ -108,7 +108,11 @@
     if hash['bindDNpassword']
       password = CernerSplunk::DataBag.load hash['bindDNpassword'], default: default_coords, type: :vault
       fail 'Password must be a String' unless password.is_a?(String)
-      hash['bindDNpassword'] = password
+      ruby_block 'set bindDNpassword' do
+        block do
+          hash['bindDNpassword'] = CernerSplunk.splunk_encrypt_password password, ::File.join(node['splunk']['home'], 'etc/auth/splunk.secret')
```

Comment:
Should someone invoking the method have to know the location of the splunk secret file or can we infer it?